CAE/CorporateEvents/CorporateActions.rdf

```diff
@@ -230,21 +298,23 @@
+		<skos:definition xml:lang="en">action that provides information regarding, for example, a corporate action, or some other activity</skos:definition>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-cae-ce-act;OrganizationNameChange">
+	<owl:Class rdf:about="&fibo-cae-ce-act;OrganizationAddressChange">
 		<rdfs:subClassOf rdf:resource="&fibo-cae-ce-act;CorporateInformationAction"/>
-		<rdfs:label xml:lang="en">organization name change</rdfs:label>
-		<skos:definition xml:lang="en">The issuing company changes it&apos;s name. Event shows the change from old name to new name and may involve surrendering physical shares with the old name to the registrar. SWIFT = NAME</skos:definition>
+		<rdfs:label xml:lang="en">organization address change</rdfs:label>
+		<skos:definition xml:lang="en">information action that provides details of address changes for a legal entity</skos:definition>
+		<skos:note xml:lang="en">Address changes may impact customer 360 and/or securities master data management, including but not limited to where shares are registered and by whom.</skos:note>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-cae-ce-act;PlaceOfIncorporationChange">
+	<owl:Class rdf:about="&fibo-cae-ce-act;OrganizationNameChange">
 		<rdfs:subClassOf rdf:resource="&fibo-cae-ce-act;CorporateInformationAction"/>
-		<rdfs:label xml:lang="en">place of incorporation change</rdfs:label>
-		<skos:definition xml:lang="en">Changes in the state of incorporation for US companies and changes in the place of incorporation for foreign companies. Where shares need to be registered following the incorporation change, the holder(s) may have to elect the registrar. SWIFT = PLAC</skos:definition>
+		<rdfs:label xml:lang="en">organization name change</rdfs:label>
+		<skos:definition xml:lang="en">information action that provides details of name changes for a legal entity</skos:definition>
+		<skos:note xml:lang="en">Name changes may include legal name changes as well as &apos;doing business as&apos;, and other operational names for an organization, some of which may impact customer 360 and/or securities master data management.</skos:note>
```

Comment:
That's covered by the legal form action not this one, I think, rather than a name change, but I don't know how people use these in practice.  It would be interesting to have the list of actions that were sent out by SWIFT when Google stock became Alphabet stock, but I have no idea who might retain that information for us to use to model that example.  Presumably there would be a slew of actions, including but not limited to the name change.